app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt

```diff
@@ -94,10 +97,22 @@ class OverdueListViewHolder(
     actionsContainer.visibility = if (isVisible) View.GONE else View.VISIBLE
   }
 
+  fun handlePhoneNumberViewVisibility() {
```

Comment:
I see you've named this function on the lines of `handleBottomLayoutVisibility()`. `handle` is a very generic name and it does not indicate anything about its functionality. When someone sees this function's name, they should immediately be able to understand what will happen when it's called.

Reply:
Can you also rename `handleBottomLayoutVisibility()` along with this?